From d4aa8e648ef878f1f59cd3d0e859b0365930603d Mon Sep 17 00:00:00 2001
From: Jan Luebbe <jlu@pengutronix.de>
Date: Thu, 1 Feb 2018 18:44:35 +0100
Subject: [PATCH 03/44] uncrustify: add configuration and helper script

This initial configuration tries to keep the changes as small as
possible.

Signed-off-by: Jan Luebbe <jlu@pengutronix.de>
---
 .uncrustify.cfg | 13 +++++++++++++
 uncrustify.sh   |  7 +++++++
 2 files changed, 20 insertions(+)
 create mode 100644 .uncrustify.cfg
 create mode 100755 uncrustify.sh

diff --git a/.uncrustify.cfg b/.uncrustify.cfg
new file mode 100644
index 0000000..254201d
--- /dev/null
+++ b/.uncrustify.cfg
@@ -0,0 +1,13 @@
+# Uncrustify-0.66.1-2-f9c285db
+string_replace_tab_chars        = true
+sp_assign                       = add
+sp_func_proto_paren             = remove
+sp_func_def_paren               = remove
+sp_func_call_paren              = remove
+sp_cond_ternary_short           = remove
+indent_func_call_param          = true
+indent_func_param_double        = true
+indent_switch_case              = 8
+align_keep_tabs                 = true
+# option(s) with 'not default' value: 10
+#
diff --git a/uncrustify.sh b/uncrustify.sh
new file mode 100755
index 0000000..78cdccf
--- /dev/null
+++ b/uncrustify.sh
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+set -ex
+
+cd `dirname $0`
+
+uncrustify -c .uncrustify.cfg -l C --replace src/*.c include/*.h test/*.[ch]
-- 
2.14.1

